README.md

```diff
@@ -1,6 +1,7 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+In order to use this repository you must set an environment variable of PGDATABASE in an env.development file. 
```

Comment:
Don't forget we need the .env for the test DB too. Anyone downloading your repo  to look at it may very well want to run the tests which will use this DB as you will have been using it for your tests